scripts/us_bls/jolts/bls_jolts.py

```diff
@@ -184,17 +226,35 @@ def row_to_stat_var(row):
         seasonal_adjustment = row['seasonal_adjustment']
 
         return (
-            f"dcs:{base_stat_var}_NAICS_{industry_code}_{seasonal_adjustment}")
+            f"dcs:{base_stat_var}_NAICS{industry_code}_{seasonal_adjustment}")
 
     # Build map to Statistical Variable.
     jolts_df['seasonal_adjustment'] = jolts_df['seasonal'].apply(
         lambda adj: "Adjusted" if adj == "S" else "Unadjusted")
     jolts_df['StatisticalVariable'] = jolts_df.apply(row_to_stat_var, axis=1)
+    for old, new in _dcid_map.items():
+        jolts_df['StatisticalVariable'] = jolts_df[
+            'StatisticalVariable'].str.replace(old, new, regex=False)
     jolts_df['Value'] = jolts_df['value']
 
     return jolts_df, schema_mapping
 
 
+def remap_dcid():
```

Comment:
Pls add a block comment on what this function is doing.
Since it is only calling print(0 but not returning anything, is this just to print the mcf on console fro debugging or is the output redirected to a file?